lib/library-functions.sh

```diff
@@ -307,6 +307,15 @@ set -o errexit  ## set -e : exit the script if any statement returns a non-true
 # shellcheck source=lib/functions/compilation/atf.sh
 source "${SRC}"/lib/functions/compilation/atf.sh
 
+# no errors tolerated. invoked before each sourced file to make sure.
```

Comment:
# This file is/was autogenerated by lib/tools/gen-library.sh; don't modify manually

Was this generated or edited?

Reply:
Nice catch. That was edited. Now its generated